src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -74,8 +74,10 @@ public Cookie getCurrCookie(HttpServletRequest request) {
   }
 
   // logs in user and adds cookie with value of String userID to response
-  public void loginUser(HttpServletRequest request, HttpServletResponse response) {
+  public long loginUser(HttpServletRequest request, HttpServletResponse response) {
     currCookie = getCurrCookie(request);
+    long userId = -1l;
```

Comment:
`NO_LOGGED_IN_USER` (constant probably needs to be a long since the majority of the checks against it are for long comparisons)